src/runtimes.ts

```diff
@@ -32,7 +34,7 @@ export const funCacheDir = XDGAppPaths('co.zeit.fun').cache();
 
 function createRuntime(
 	runtimes: Runtimes,
-	name: string,
+	name: RuntimeLiteral,
```

Comment:
Hmm, I'm not sure we want to change this yet as you mentioned about type casting.

Let's do that in separate PR.

Reply:
:+1: I'll remove that from this PR & make a new one